crypto/rand/rand_lib.c

```diff
@@ -189,6 +194,7 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
         }
 
     } else {
+#if 0
```

Comment:
add a comment on why this was commented out please

Reply:
OK.